server/controllers/download-by-content-id.js

```diff
@@ -81,25 +81,25 @@ module.exports = exports = async (req, res, next) => {
 
 	dl.on('complete', (state, status) => {
 		if (state === 'complete') {
-			log.count(`${articleOrArchive.toLowerCase()}-download-complete`)
+			log.count(`${articleOrArchive.toLowerCase()}-download-complete`);
 		}
 		res.status(status);
 	});
 
-	dl.on('end', () => {
-		log.debug(`Download${articleOrArchive}End => ${content.id} in ${Date.now() - dl.START}ms`);
-
-		if (dl.cancelled !== true) {
-			res.end();
-			next();
-		}
-	});
-
 	dl.on('cancelled', () => {
 		next();
 	});
 
 	if (articleOrArchive === 'Archive') {
+		dl.on('end', () => {
```

Comment:
I've tested that code (`dl.on`) and it only runs for archives (a.k.a videos).  So I'm moving to the `if` clause